test/ts-scenario/steps/cli.ts

```diff
@@ -1,26 +1,27 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
 /**
  * SPDX-License-Identifier: Apache-2.0
  */
 
 'use strict';
 
-import { Constants } from './constants';
+import {Constants} from './constants';
 import * as Channel from './lib/channel';
 import * as Contract from './lib/contract';
 import * as BaseUtils from './lib/utility/baseUtils';
-import { CommonConnectionProfileHelper } from './lib/utility/commonConnectionProfileHelper';
-import { StateStore } from './lib/utility/stateStore';
+import {StateStore} from './lib/utility/stateStore';
+
+import {Given} from 'cucumber';
 
-import { Given } from 'cucumber';
-import * as path from 'path';
 
 const stateStore: StateStore = StateStore.getInstance();
 const orgNames: string[] = ['Org1', 'Org2'];
 
-const ccpNonTls: CommonConnectionProfileHelper = new CommonConnectionProfileHelper(path.join(__dirname, '../config', 'ccp.json'), true);
-const ccpTls: CommonConnectionProfileHelper = new CommonConnectionProfileHelper(path.join(__dirname, '../config', 'ccp-tls.json'), true);
 
-Given(/^I use the cli to create and join the channel named (.+?) on the deployed network/, { timeout: Constants.STEP_MED as number }, async (channelName: string) => {
+Given(/^I use the cli to create and join the channel named (.+?) on the deployed network/, {timeout: Constants.STEP_MED as number}, async (channelName: string) => {
```

Comment:
Looks to be unnecessary code below. The function is async so can just return a value or directly throw an error